### PR TITLE
Implements demo auth buttons

### LINKED ISSUE #45: FE: Demo User
This ticket does not have a mock-up so please add in the button in the Login and Sign up pages and make sure the design is consistent with the overall look and feel.

- Create a "demo user" functionality that will be used later to allow recruiters and managers to easily test the app's features without having to sign up
- Add a button to allow for signing in as a demo user
- This is effectively a login button with a hardcoded username and password
- Show button both on signup and login pages
- In production you should manually create a user and some test data that would best show off your project. Do this in development right now to test this feature

### PR DESCRIPTION
**Pull request by EdwardOlmos**
resolves #45 - FE: Demo User

### What this PR does (required):
- Allow recruiters and managers to easily test the app's features without having to sign up by clicking hard-coded credential buttons.

### Screenshots / Videos (front-end only):
- Login Page
login_page (2)

- Sign Up Page
signup_page (2)

- After clicking 'Demo Dog Owner' or 'Demo Dog Sitter'. The expected result should be the dashboard page.
dashboard_page

### Any information needed to test this feature (required):
- Manually register these users, only for the first time.
  - Dog Owner
    - Username: Dog Owner
    - Email: dogowner@demo.com
    - Password: Test123!
  - Dog Sitter
    - Username: Dog Sitter
    - Email: dogsitter@demo.com
    - Password: Test123!
    
Walkthrough
1. Logout from any session, then visit the Login Page and click the 'Demo Dog Owner' button
2. Expect to get redirected to the dashboard page as a Dog Owner user
3. Logout from any session, then visit the Sign Up Page and click the 'Demo Dog Sitter' button
4. Expect to get redirected to the dashboard page as a Dog Sitter user

### CODE REVIEW

**Review by ezeilo-su — COMMENTED** [review]
Looks good to me!

**Review by ugur-akin — COMMENTED** [review]
Hey @EdwardOlmos , good work! Just a minor organization issue that'd be better if addressed 👍🏼 

Thanks for putting effort behind communicating your PR, it really helps! Just a couple of quick notes:
- We can opt for descriptive titles, in plain English (e.g. `Implements demo auth buttons`), the ticket and other details ideally will already be present within the PR.
- When mentioning the issue in your pull request summary, it's a good habit to use one of the supported keywords for [automatically linking your issue through Github](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). Note that this is different than connecting your issue with ZenHub. I'd recommend doing both as some envorinments will use one and some the other 👍🏼 

I also recommend taking a look at [this article](https://hugooodias.medium.com/the-anatomy-of-a-perfect-pull-request-567382bb6067) for further tips on how to communicate your PRs effectively.

Overall very good start, let's keep it up! 💯 
Cheers 🎉

**Comment by EdwardOlmos** [comment]
I updated the login and sign-up screenshots to reflect the previous change (moving and adjusting styles). I also used the `resolves` keyword for automatic issue linking 👍

**Review by ugur-akin — APPROVED** [review]
Awesome, thanks 🙏🏼  LGTM! 🚀
